### PR TITLE
[BACK-2679] Deps update

### PR DESCRIPTION
**Pull request by jh-bate**
dependency updates for audit:

- remove unused imports 
- remove yarn.lock as using npm
- run `npm outdated` and apply updates
- run `npm fix audit`
- updated node version, same as export service now

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ dist: focal
 language: node_js
 
 node_js:
-  - 14.21.2
+  - 16.20.1
```
Comment (darinkrauss) [inline .travis.yml:8]: So, `16.20.1` is the supported version of choice for the other Node.js repos?

**Review by darinkrauss — DISMISSED** [review]
LGTM!

File: .travis.yml (hunk 1 above)
Comment (jh-bate) [reply .travis.yml:8]: It was for the export service that was recently upgraded https://github.com/tidepool-org/export/blob/94988ea6eda903853967cba86942d6ac6b839d98/.travis.yml#L6

**Comment by toddkazakov** [comment]
This one can't start QA2. It's stuck in `CrashLoopBackOff`. This is the error:
```
                                                                                                                                                                      │
│ > jellyfish@0.14.0 start                                                                                                                                              │
│ > node app                                                                                                                                                            │
│                                                                                                                                                                       │
│ npm ERR! code EROFS                                                                                                                                                   │
│ npm ERR! syscall mkdir                                                                                                                                                │
│ npm ERR! path /.npm                                                                                                                                                   │
│ npm ERR! errno -30                                                                                                                                                    │
│ npm ERR! rofs EROFS: read-only file system, mkdir '/.npm'                                                                                                             │
│ npm ERR! rofs Often virtualized file systems, or other file systems                                                                                                   │
│ npm ERR! rofs that don't support symlinks, give this error.                                                                                                           │
│                                                                                                                                                                       │
│ npm ERR! Log files were not written due to an error writing to the directory: /.npm/_logs                                                                             │
│ npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal                                                                         │
│ Stream closed EOF for qa2/jellyfish-fc74c5ffb-fzt4l (jellyfish)
```

I'll remove it from QA2, because it interferes with other deployments.

**Pull request closed without merging**